advanced/sr_gui_biotac/src/sr_gui_biotac/biotac.py

```diff
@@ -284,36 +283,39 @@ def __init__(self, context):
         self._nb_electrodes = self._nb_electrodes_biotac
         self.assign_electrodes(self._nb_electrodes)
 
-        ui_file = os.path.join(rospkg.RosPack().get_path('sr_gui_biotac'),
-                               'uis', 'SrGuiBiotac.ui')
-        loadUi(ui_file, self._widget)
-        self._widget.setObjectName('SrBiotacUi')
-
-        self.timer = QTimer(self._widget)
-        # self._widget.connect(self.timer, SIGNAL("timeout()"),
-        #                     self._widget.scrollAreaWidgetContents.update)
-        self.timer.timeout.connect(self._widget.scrollAreaWidgetContents.update)
-        self._widget.scrollAreaWidgetContents.paintEvent = self.paintEvent
+        if rqt_plugin:
```

Comment:
There are three statements " if rqt_plugin:". we should joint them all in one

Reply:
I've joined two of them. Can I move subscribe to the end of the code (so that I can join the remaining two)?

Reply:
I think we can subscribe at the end. Can you test that?